DesignSafe-CI/portal
# DES-1802 - Update information about Django tests

## Overview: ##
Update information about Django testing to guide people to use pytest.

## PR Status: ##


## Related Jira tickets: ##

* [DES-1802](https://jira.tacc.utexas.edu/browse/DES-1802)

## Summary of Changes: ##
Small change to README.md.

## Testing Steps: ##
1. Run the command `docker exec -it des_django pytest designsafe`
2. Confirm that tests run.

## UI Photos:
N/A

## Notes: ##
